tox.ini

```diff
@@ -20,7 +20,7 @@ commands =
 [testenv:lint]
 commands =
    pylint src --disable=missing-module-docstring,invalid-name,too-few-public-methods,\
-   redefined-builtin,too-many-lines src
+   redefined-builtin,too-many-lines,no-value-for-parameter src

```

Comment:
`no-value-for-parameter` is an important check ignoring which for whole application can break things. better ignore on local scope where main is called.

`main() # pylint: disable=no-value-for-parameter`

Reply:
lint gets perfect score without `too-many-lines`. better remove it. probably some refactoring fixed too many line issues at some point.

Reply:
Done.